src/signals/IncidentMap/components/utils/update-filter-category.ts

```diff
@@ -0,0 +1,16 @@
+import type { Filter } from '../../types'
+
+export const updateFilterCategory = (
+  categoryName: string,
+  filters: Filter[]
+) => {
+  return filters.map((filter) => {
```

Comment:
filter map 😃you could however remove the round brackets.

Reply:
Prettier adds them, turning the rule off doesn't seem to work. Is it really bothering?

Reply:
no, don't really care! my editor wasn't doing anything on save, didn't saw the prettier hook kicking in